pkg/api/volume_req.go

```diff
@@ -20,6 +20,12 @@ type SubvolReq struct {
 }
 
 // VolCreateReq represents a Volume Create Request
+/*supported Flags
```

Comment:
Add `:` or a `-` after the flag and before it's description

Reply:
done